manifests/certificate/x509.pp

```diff
@@ -97,6 +101,7 @@
   $password = undef,
   $force = true,
   $cnf_tpl = 'openssl/cert.cnf.erb',
+  $encrypted = true,
```

Comment:
Are you sure? Currently your default codebase will encrypt them and my pull request is to allow for unencrypted. Having the default value be true (ie encrypted) preserves your existing behaviour. By having it changed to false (unencrypted) that would suddenly change all your users configurations on them unexpectedly.